docs/development_environment.md

```diff
@@ -46,130 +44,52 @@ It is also possible to set up a more traditional development environment. See th
 Install the core dependencies.
 
 ```shell
-$ sudo apt-get install python3-pip python3-dev python3-venv
-```
-
-In order to run `script/setup` below you will need some more dependencies.
-
-```shell
-$ sudo apt-get install autoconf libssl-dev libxml2-dev libxslt1-dev libjpeg-dev libffi-dev libudev-dev zlib1g-dev pkg-config
-$ sudo apt-get install -y libavformat-dev libavcodec-dev libavdevice-dev libavutil-dev libswscale-dev libavresample-dev libavfilter-dev
+sudo apt-get install python3-pip python3-dev python3-venv autoconf libssl-dev libxml2-dev libxslt1-dev libjpeg-dev libffi-dev libudev-dev zlib1g-dev pkg-config libavformat-dev libavcodec-dev libavdevice-dev libavutil-dev libswscale-dev libavresample-dev libavfilter-dev
```

Comment:
I think we should add ffmpeg as well. The camera/stream integration is kinda hardwired now because of Alexa and Cloud. It is really likely one would need it.

Reply:
```suggestion
sudo apt-get install python3-pip python3-dev python3-venv autoconf libssl-dev libxml2-dev libxslt1-dev libjpeg-dev libffi-dev libudev-dev zlib1g-dev pkg-config libavformat-dev libavcodec-dev libavdevice-dev libavutil-dev libswscale-dev libavresample-dev libavfilter-dev ffmpeg
```